Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN python -c "import duckdb; duckdb.connect().sql(\"INSTALL httpfs\");"
 
 FROM ubuntu:22.04 AS runner-image
 
-RUN apt-get update && apt-get install --no-install-recommends -y python3.11 python3.11-venv && \
+RUN apt-get update && apt-get install --no-install-recommends -y python3.11 python3.11-venv msodbcsql18 && \
```

Comment:
Why is this necessary? Can you share a little bit on that?

Reply:
We do not use the Dockerfile for test execution. We run the tests with pytest locally, and we use github ci to execute them on CI. Testcontainers is only used to provision databases so we test against the actual thing. 

We should install that driver in the github workflow file instead. And we should disable the test when not in CI as it would otherwise force everybody to install those drivers on their machines to male the tests run.

Reply:
These two statements seem to be conflicting. If you want to execute that test and have it succeed it requires the driver. As for the CI I can add it there as well, but as it stands I'm unsure what the consensus is for having it in the Dockerfile as well

Reply:
I think both comments are valid. We need the lib in the Dockerfile for `datacontract test` command, but we also need the lib in the CI pipeline so we can run `pytest` in GitHub actions as well as run the test locally.

Reply:
Added to both areas

Reply:
Updated to skip the test outside of CI/CD and likewise removed from dockerfile